### ISSUE TITLE
Support new functionality in Chalk 2 in <Text>

### ISSUE THREAD

**Issue by vadimdemedes** [vadimdemedes/ink#19]
Like `rgb()` or `hex()`.

**Comment by vadimdemedes** [comment]
Pushed https://github.com/vadimdemedes/ink/commit/023e8d8f99eed2c0e4074e12e9b925ecd333f4ae to implement this. Keeping the issue open until release is out.

Examples on how to use all Chalk 2 features (will be added later to the readme):

```js
<Text rgb={[40, 42, 54]}>Test</Text>
//=> chalk.rgb(40, 42, 54)('Test');

<Text keyword="orange">Test</Text>
//=> chalk.keyword('orange')('Test');

<Text blue>Test</Text>
//=> chalk.blue('Test');
```

**Comment by vadimdemedes** [comment]
Landed in `0.3.0`.
